features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,801 @@
+                        enableSMSBasedRecovery: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_SMS_EXPIRY_TIME) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        smsOtpExpiryTime: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_UPPERCASE) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseUppercase: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_LOWERCASE) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseLowercase: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_NUMERIC) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseNumeric: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_LENGTH) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        smsOtpLength: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_MAX_RESEND_COUNT) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        maxResendCount : property.value
+                    };
+                }  else if (property.name === ServerConfigurationsConstants.RECOVERY_MAX_FAILED_ATTEMPTS_COUNT) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        maxFailedAttemptCount : property.value
+                    };
+                }
+            }
+        });
+        setInitialConnectorValues(resolvedInitialValues);
+        setIsEmailRecoveryEnabled(resolvedInitialValues?.enableEmailBasedRecovery);
+        setIsSMSRecoveryEnabled(resolvedInitialValues?.enableSMSBasedRecovery);
+        setIsUpperCaseEnabled(resolvedInitialValues?.passwordRecoveryOtpUseUppercase);
+        setIsLowerCaseEnabled(resolvedInitialValues?.passwordRecoveryOtpUseLowercase);
+        setIsNumericEnabled(resolvedInitialValues?.passwordRecoveryOtpUseNumeric);
+    }, [ initialValues ]);
+
+    /**
+     * Validate input data.
+     *
+     * @param values - Form values.
+     * @returns Form validation.
+     */
+    const validateForm = (values: any):
```

Comment:
use an interface in place of any

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/0dd77c8c16b11b96a91e6b1331294e3bb8560a5b